### PR TITLE
Tech extensions: remove feature toggle leftovers

### PR DESCRIPTION
**Pull request by lburgazzoli**

### CODE REVIEW

**Comment by lburgazzoli** [comment]
@rhuss I think you need to approve it again after fixing some issues

File: app/runtime/runtime/src/main/java/io/syndesis/integration/runtime/stephandlers/FunctionHandler.java [hunk 1]
```diff
@@ -58,6 +59,7 @@ public ProcessorDefinition handle(Function step, ProcessorDefinition route, Synd
         Map<String, Object> headers = step.getProperties();
         if (ObjectHelper.isNotEmpty(headers)) {
             options = headers.entrySet().stream()
+                .filter(entry -> Objects.nonNull(entry.getValue()))
```
Comment (zregvart) [inline app/runtime/runtime/src/main/java/io/syndesis/integration/runtime/stephandlers/FunctionHandler.java]: This seems unrelated to tech extensions feature toggle

Comment (lburgazzoli) [reply app/runtime/runtime/src/main/java/io/syndesis/integration/runtime/stephandlers/FunctionHandler.java]: uh yeah, that was supposed to be included in a different pr
